bin/auth-api/src/services/users.service.ts

```diff
@@ -39,7 +39,7 @@ export async function getUserById(id: UserId) {
 }
 export type UserWithTosStatus = NonNullable<Awaited<ReturnType<typeof getUserById>>>;
 
-export async function getUserByAuth0Id(auth0Id: string) {
+export async function getUserByAuth0Details(auth0Id: string) {
```

Comment:
why rename? In this case we are passing in the auth0 id, not the full blob of details?

Reply:
sorry, that's a hangup on that I was also going to pass email - will revert that!